src/core/lightning/index.ts

```diff
@@ -612,19 +602,22 @@ export const LightningMixin = (superclass) =>
 
           // success
           await ledger.settleLndPayment(id)
-          const paymentResult = await paymentPromise
 
           if (!feeKnownInAdvance) {
             await this.recordFeeDifference({
-              paymentResult,
+              paymentResult: { safe_fee: result.roundedUpFee },
               max_fee,
               id,
               related_journal: entry.journalId,
             })
           }
 
           lightningLogger.info(
-            { success: true, paymentResult, ...metadata },
+            {
+              success: true,
+              paymentResult: { safe_fee: result.roundedUpFee },
```

Comment:
with the previous code we got the full response so not sure if is necessary do the wrapping here, maybe just the payment id and preimage

Reply:
Yup this is one of the open questions that I swapped in here because this is how it's being done currently in the new use-case based on how we're doing the types.

Do you mean we should add the payment id and preimage to the `PayInvoiceResult` type too?

Reply:
I just realised, should we be logging things like the `preimage` at all?